roles/k8s_best_practices_certsuite/tasks/teardown.yml

```
@@ -1,14 +1,16 @@
 ---
 # Always submit these files to DCI:
-# - Configuration files - tnf_config.yml
+# - Configuration files - tnf_config.yml (or certsuite_config.yml)
 # - Execution logs - certsuite-stdout.log and certsuite.log, if they exist
 - name: Copy config and log files related to certsuite execution
+  vars:
+    kbpc_config_file_name: "{{ (kbpc_version == 'HEAD') | ternary('certsuite_config', 'tnf_config') }}"
   ansible.builtin.copy:
     src: "{{ item }}"
     dest: "{{ kbpc_log_path }}"
     mode: "0750"
   with_fileglob:
-    - "{{ kbpc_certsuite_dir }}/config_files/tnf_config.yml"
+    - "{{ kbpc_certsuite_dir }}/config_files/{{ kbpc_config_file_name }}.yml"
```

We can still make use of globbing to find the config file, if the other has been removed:
```suggestion
    - "{{ kbpc_certsuite_dir }}/config_files/*_config.yml"
```

---

ack